lib/local/platform/windows.js

```diff
@@ -36,6 +44,15 @@ module.exports = {
     pathQuery: 'dir /s /b iexplore.exe',
     cwd: cwd
   },
+  edge: {
+    defaultLocation: path.join(microsoftEdgeDirectory, 'MicrosoftEdge.exe'),
+    getCommand: function(browser, url) {
+      return 'start /wait microsoft-edge:' + url;
```

Comment:
This command correctly starts up the browser and I see the `Hello, world!` in the browser but isn't shutdown correctly. Any tips on how to make that happen correctly? I'm trying to figure out the launchpad code. It might be that the `imageName` needs to be `MicrosoftEdge.exe`?